<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WhetherWindow</class>
 <widget class="QMainWindow" name="WhetherWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WhetherWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="main.qrc">
    <normaloff>:/image/_displayLogo.png</normaloff>:/image/_displayLogo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="updateTime">
    <property name="geometry">
     <rect>
      <x>142</x>
      <y>500</y>
      <width>220</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="airCondition">
    <property name="geometry">
     <rect>
      <x>142</x>
      <y>453</y>
      <width>220</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>30</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="highestTemp">
    <property name="geometry">
     <rect>
      <x>142</x>
      <y>170</y>
      <width>220</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="whether">
    <property name="geometry">
     <rect>
      <x>142</x>
      <y>122</y>
      <width>220</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="windPower">
    <property name="geometry">
     <rect>
      <x>142</x>
      <y>406</y>
      <width>220</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>30</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="winds">
    <property name="geometry">
     <rect>
      <x>142</x>
      <y>311</y>
      <width>220</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="windVector">
    <property name="geometry">
     <rect>
      <x>142</x>
      <y>359</y>
      <width>220</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_Title">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>花卉信息管理系统天气助手——微特</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Area">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>地区：</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>36</x>
      <y>121</y>
      <width>101</width>
      <height>421</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_Weather">
       <property name="text">
        <string>天气状况：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_HighestTemp">
       <property name="text">
        <string>最高温度：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_LowestTemp">
       <property name="text">
        <string>最低温度：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_CurrentTemp">
       <property name="text">
        <string>现在温度：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_Winds">
       <property name="text">
        <string>刮风情况：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_WindVector">
       <property name="text">
        <string>风向：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_WindPower">
       <property name="text">
        <string>风力：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_AirCondition">
       <property name="text">
        <string>空气质量：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_UpdateTime">
       <property name="text">
        <string>更新时间：</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="currentTemp">
    <property name="geometry">
     <rect>
      <x>142</x>
      <y>264</y>
      <width>220</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="lowestTemp">
    <property name="geometry">
     <rect>
      <x>142</x>
      <y>217</y>
      <width>220</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="queryBtn">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>70</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>查询</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cityComboBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>武汉</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>黄冈</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>荆州</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>宜昌</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>恩施</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>十堰</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>神农架</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>随州</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>荆门</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>天门</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>仙桃</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>潜江</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>襄阳</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>鄂州</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>孝感</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>黄石</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>咸宁</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>北京</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>上海</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>天津</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>重庆</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>哈尔滨</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>长春</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>沈阳</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>呼和浩特</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>石家庄</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>太原</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>西安</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>济南</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>乌鲁木齐</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>拉萨</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>西宁</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>兰州</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>银川</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>郑州</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>南京</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>杭州</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>合肥</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>福州</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>南昌</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>长沙</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>贵阳</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>成都</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>广州</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>昆明</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>南宁</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>海口</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>香港</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>澳门</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>台北县</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>408</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="main.qrc"/>
 </resources>
 <connections/>
</ui>
